src/components/header/index.tsx

```diff
@@ -2,39 +2,23 @@ import Image from "next/image";
 import Link from "next/link";
 import { urlFor } from "@/app/actions/sanityClient";
 
-const Header = async ({ globalConfig }: any) => {
+const Header = ({ globalConfig }: any) => {
   return (
-    <header className="govuk-header" data-module="govuk-header">
-      <div className="govuk-header__container govuk-width-container">
-        <div className="govuk-header__logo">
-          <Link
-            href="/"
-            style={{
-              color: "white",
-              textDecoration: "none",
-              fontWeight: "bold",
-            }}
-          >
-            <span
-              className="govuk-header__logotype"
-              style={{ display: "flex", alignItems: "center", gap: "2px" }}
-            >
-              {globalConfig?.logo !== undefined ? (
-                <Image
-                  width={100}
-                  height={35}
-                  alt=""
-                  src={urlFor(globalConfig?.logo)?.url()}
-                />
-              ) : (
-                ""
-              )}
-              <span className="govuk-header__logotype-text" role="definition">
-                Planning applications
-              </span>
-            </span>
-          </Link>
-        </div>
+    <header className="dsn-header">
+      <div className="dsn-header__content">
+        {globalConfig?.logo && (
+          <div className="dsn-header__logo">
+            <Link href="/">
```

Comment:
I think we might need to check with the team, but I believe the link is supposed to be trigger-able on the whole header, or at least both the council logo and the 'Planning applications' title

Reply:
Have checked with Bobbu and the links are all good - also boring spoiler alert later on the header gets wrapped in the page container! 🎉